### PR TITLE
Move a part of AccessibleObject classes to separate files

### PR DESCRIPTION
**Pull request by vladimir-krestov**
## Proposed changes

- Move several AccessibleObject classes to new separate files
- There are no functional changes

## Customer Impact

- No

## Regression? 

- No

## Risk

- Minimal

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/4066)

### CODE REVIEW

File: src/System.Windows.Forms/src/System/Windows/Forms/DomainUpDown.DomainUpDownAccessibleObject.cs [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+using static Interop;
+
+namespace System.Windows.Forms
+{
+    public partial class DomainUpDown
+    {
+        public class DomainUpDownAccessibleObject : ControlAccessibleObject
+        {
+            private DomainItemListAccessibleObject itemList;
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/DomainUpDown.DomainUpDownAccessibleObject.cs:15]: Please fix the naming

File: src/System.Windows.Forms/src/System/Windows/Forms/ToolStrip.ToolStripAccessibleObject.cs [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+using System.Drawing;
+using static Interop;
+
+namespace System.Windows.Forms
+{
+    public partial class ToolStrip
+    {
+        public class ToolStripAccessibleObject : ControlAccessibleObject
+        {
+            private readonly ToolStrip owner;
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/ToolStrip.ToolStripAccessibleObject.cs:17]: ditto

File: src/System.Windows.Forms/src/System/Windows/Forms/ToolStripDropDownButton.ToolStripDropDownButtonAccessibleObject.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+namespace System.Windows.Forms
+{
+    public partial class ToolStripDropDownButton
+    {
+        /// <summary>
+        ///  An implementation of Accessibleobject for use with ToolStripDropDownButton
+        /// </summary>
+        internal class ToolStripDropDownButtonAccessibleObject : ToolStripDropDownItemAccessibleObject
+        {
+            private readonly ToolStripDropDownButton ownerItem;
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/ToolStripDropDownButton.ToolStripDropDownButtonAccessibleObject.cs:18]: ditto

File: src/System.Windows.Forms/src/System/Windows/Forms/ToolStripGrip.ToolStripGripAccessibleObject.cs [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+using static Interop;
+
+namespace System.Windows.Forms
+{
+    internal partial class ToolStripGrip
+    {
+        internal class ToolStripGripAccessibleObject : ToolStripButtonAccessibleObject
+        {
+            private string stockName;
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/ToolStripGrip.ToolStripGripAccessibleObject.cs:15]: ditto.
Please rename to something a bit more meaningful while at it.

File: src/System.Windows.Forms/src/System/Windows/Forms/ToolStripLabel.ToolStripLabelAccessibleObject.cs [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+using static Interop;
+
+namespace System.Windows.Forms
+{
+    public partial class ToolStripLabel
+    {
+        internal class ToolStripLabelAccessibleObject : ToolStripItemAccessibleObject
+        {
+            private readonly ToolStripLabel ownerItem;
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/ToolStripLabel.ToolStripLabelAccessibleObject.cs:15]: ditto

File: src/System.Windows.Forms/src/System/Windows/Forms/ToolStripLabel.ToolStripLabelAccessibleObject.cs [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+            public ToolStripLabelAccessibleObject(ToolStripLabel ownerItem) : base(ownerItem)
+            {
+                this.ownerItem = ownerItem;
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/ToolStripLabel.ToolStripLabelAccessibleObject.cs:19]: Remove `this.`

File: src/System.Windows.Forms/src/System/Windows/Forms/ToolStripMenuItem.ToolStripMenuItemAccessibleObject.cs [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+namespace System.Windows.Forms
+{
+    public partial class ToolStripMenuItem
+    {
+        /// <summary>
+        ///  An implementation of AccessibleChild for use with ToolStripItems
+        /// </summary>
+        internal class ToolStripMenuItemAccessibleObject : ToolStripDropDownItemAccessibleObject
+        {
+            private readonly ToolStripMenuItem ownerItem;
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/ToolStripMenuItem.ToolStripMenuItemAccessibleObject.cs:18]: ditto

File: src/System.Windows.Forms/src/System/Windows/Forms/ToolStripOverflowButton.ToolStripOverflowButtonAccessibleObject.cs [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+using static Interop;
+
+namespace System.Windows.Forms
+{
+    public partial class ToolStripOverflowButton
+    {
+        internal class ToolStripOverflowButtonAccessibleObject : ToolStripDropDownItemAccessibleObject
+        {
+            private string stockName;
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/ToolStripOverflowButton.ToolStripOverflowButtonAccessibleObject.cs:15]: ditto

File: src/System.Windows.Forms/src/System/Windows/Forms/ToolStripStatusLabel.ToolStripStatusLabelAccessibleObject.cs [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+using static Interop;
+
+namespace System.Windows.Forms
+{
+    public partial class ToolStripStatusLabel
+    {
+        internal class ToolStripStatusLabelAccessibleObject : ToolStripLabelAccessibleObject
+        {
+            private readonly ToolStripStatusLabel ownerItem;
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/ToolStripStatusLabel.ToolStripStatusLabelAccessibleObject.cs:15]: ditto

**Review by RussKie — CHANGES_REQUESTED** [review]

File: src/System.Windows.Forms/src/System/Windows/Forms/ToolStripGrip.ToolStripGripAccessibleObject.cs (hunk 4 above)
Comment (vladimir-krestov) [reply src/System.Windows.Forms/src/System/Windows/Forms/ToolStripGrip.ToolStripGripAccessibleObject.cs:15]: It was a default name value.
But actually, there was a very strange implementation, so I implemented it like this: 
image

File: src/System.Windows.Forms/src/System/Windows/Forms/ToolStripOverflowButton.ToolStripOverflowButtonAccessibleObject.cs (hunk 8 above)
Comment (vladimir-krestov) [reply src/System.Windows.Forms/src/System/Windows/Forms/ToolStripOverflowButton.ToolStripOverflowButtonAccessibleObject.cs:15]: Just removed this field and implement Name property like this:
image

File: src/System.Windows.Forms/src/System/Windows/Forms/DomainUpDown.DomainUpDownAccessibleObject.cs (hunk 1 above)
Comment (vladimir-krestov) [reply src/System.Windows.Forms/src/System/Windows/Forms/DomainUpDown.DomainUpDownAccessibleObject.cs:15]: Also, fixed `_owner` naming here and in `NumericUpDownAccessibleObject`

**Review by RussKie — APPROVED** [review]

**Pull request merged**